projects/igniteui-angular/src/lib/core/utils.ts

```diff
@@ -98,10 +98,38 @@ export function isObject(value: any): boolean {
  * @returns true if provided variable is Date
  * @hidden
  */
-export function isDate(value: any) {
+export function isDate(value: any): boolean {
     return Object.prototype.toString.call(value) === '[object Date]';
```

Comment:
Shouldn't this be `value instanceof Date'?

Reply:
Sounds cleaner to me too.